editor/filesystem_dock.cpp

```diff
@@ -1447,6 +1447,15 @@ void FileSystemDock::_files_list_rmb_select(int p_item, const Vector2 &p_pos) {
 	file_options->popup();
 }
 
+void FileSystemDock::_rmb_pressed(const Vector2 &p_pos) {
+	folder_options->clear();
+	folder_options->set_size(Size2(1, 1));
+
+	folder_options->add_item(TTR("New Folder.."), FOLDER_NEW_FOLDER);
```

Comment:
Shouldn't it be `FILE_NEW_FOLDER`?

Reply:
Well, thought that way too, but works fine even with FOLDER_NEW_FOLDER.

Reply:
Oh nevermind, both options exist indeed.